CONTRIBUTING.md

```diff
@@ -2,37 +2,35 @@
 
 
 KodaDot NFT gallery has a plan to be community-owned.
-We are welcoming community contribution from you.
-Take a sneak peek on [good first issues](https://github.com/kodadot/nft-gallery/contribute), comment and make PR.
-If everything goes well, chances that you will be **rewarded are high**. We might give retro-active bounties where bounty label wasn't present if we like your commit.
+We are welcoming community contributions from you.
+Take a sneak peek on [good first issues](https://github.com/kodadot/nft-gallery/contribute), comment, and make PR.
+If everything goes well, chances that you will be **rewarded are high**. We might give retro-active bounties where the bounty label wasn't present if we like your commit.
 
-**For better coordination, please join our [Development channel Anteriors on Discord](https://discord.gg/KkctBVQ8kP)**
+**For better coordination, please join our [Development channel (#garage) on Discord](https://discord.gg/KkctBVQ8kP)**
 
 
 ## Getting started
 
 Before you being:
 
-- Site is powered by **Node.js.** Check to see if you're on the version of **Node.js we support.**
+- As the development tool, we use **Node.js.** Check to see if you're on the version of **Node.js we support.**
```

Comment:
```suggestion
- We utilize **Node.js** as a development tool. To avoid potential compatibility issues, check if you're on the version of **Node.js we support.**
```